.changeset/heavy-colts-approve.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/backend-common': minor
```

Comment:
think that these are still `patch` changes though not `minor` since we are `<1.0.0`

Reply:
of course! updated.